### ISSUE TITLE
Peertube should not serve remote avatars as lazy-static

### ISSUE THREAD

**Issue by gianmarcogg03** [NixOS/nixpkgs#245567]
### Describe the bug
When having Peertube autoconfigure nginx, the nginx configuration it generates serves remote avatars as `lazy-static` when the [official nginx configuration for Peertube](https://github.com/Chocobozzz/PeerTube/blob/develop/support/nginx/peertube) does not do that, the developer mentioned it in [this other issue](https://github.com/Chocobozzz/PeerTube/issues/5895#issuecomment-1651061613).

### Steps To Reproduce
1. Use the option `configureNginx`.
2. Check remote avatars in the notifications or the subscriptions list.

### Expected behavior
The avatars should load.

### Screenshots
The avatars do not load and empty frames are shown in their place.
![image](https://github.com/NixOS/nixpkgs/assets/28053717/7342b799-727e-4628-9c51-aebc3df1309b)
![image](https://github.com/NixOS/nixpkgs/assets/28053717/75a257d7-08dd-47ea-89ab-0903c9692483)

### Notify maintainers
@immae @Izorkin @mohe2015 @stevenroose 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.
```console
 - system: `"x86_64-linux"`
 - host os: `Linux 6.3.12, NixOS, 23.05 (Stoat), 23.05.2209.ac1acba43b2`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - channels(root): `"nixos-23.05"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by Izorkin** [comment]
I'll try to update the nginx configuration a little later on.

**Comment by gianmarcogg03** [comment]
Any news on this?

**Comment by Izorkin** [comment]
I'll be able to do it next week.

**Comment by Izorkin** [comment]
@gianmarcogg03 https://github.com/NixOS/nixpkgs/pull/258068

**Comment by gianmarcogg03** [comment]
I'm on NixOS 23.05 which still has PeerTube 5.1.0, while your fixed version (5.2.1) is in the master/unstable branch. I guess we have to wait for NixOS 23.11 to come out unless someone backports the fix to 23.05 which I think is unlikely.

**Comment by gianmarcogg03** [comment]
Just updated to NixOS 23.11 and the problem is now fixed. Thank you!
